pilot/cmd/istioctl/gendeployment/helm.go

```diff
@@ -40,6 +40,8 @@ global:
    proxy_debug: {{ .Debug }}
    pilot_hub: {{ .Hub }}
    pilot_tag: {{ .PilotTag }}
+	 hyperkube_hub: {{ .HiperkubeHub }}
```

Comment:
Un-indent this.

Reply:
I think is github conmpare issue, can you check locally if you see the bad indentation too? locally I don't have it

Reply:
Oh, tab-vs-spaces issue, I think! It shows up as a tab for me. YAML should always use spaces.

Reply:
thought I have spaces as default, indeed that's the reason

Reply:
The fact that you're in a `.go` file probably threw your editor off, since golang uses tabs.